src/main/java/longah/commands/filter/FilterDateTimeCommand.java

```diff
@@ -0,0 +1,51 @@
+package longah.commands.filter;
+
+import longah.commands.Command;
+import longah.exception.ExceptionMessage;
+import longah.exception.LongAhException;
+import longah.handler.UI;
+import longah.node.Group;
+import longah.util.TransactionList;
+
+public class FilterDateTimeCommand extends Command {
+    /**
+     * Constructor for FilterDateTimeCommand.
+     *
+     * @param commandString The command string.
+     * @param taskExpression The task expression.
+     */
+    public FilterDateTimeCommand(String commandString, String taskExpression) {
+        super(commandString, taskExpression);
+    }
+
+    /**
+     * Executes the corresponding filter datetime command based on the format of the user task input.
+     *
+     * @param group The group to execute the command on.
+     * @throws LongAhException If the taskExpression for the date times to search is in the wrong format
+     */
+    public void execute(Group group) throws LongAhException {
+        TransactionList transactions = group.getTransactionList();
+        String message;
+        if (!(taskExpression.contains("b/") || taskExpression.contains("a/"))) {
```

Comment:
Am struggling to understand what's going on here, could you explain the expected behavior